test/print.jl

```diff
@@ -202,7 +202,17 @@ end
     end
 end
 
-function printing_test(ModelType::Type{<:JuMP.AbstractModel})
+"""
+    printing_test(ModelType::Type{<:JuMP.AbstractModel},
+                  moi_backend::Bool)
+
+Test printing of models of type `ModelType`. The expected model printing depends
+on `moi_backend` which indicates whether the model is stored in an MOI backend
+or is stored in its JuMP form (i.e. as `AbstractVariable`s,
+`AbstractConstraint`s and `AbstractJuMPScalar` for the objective function).
+"""
+function printing_test(ModelType::Type{<:JuMP.AbstractModel},
+                       moi_backend::Bool)
```

Comment:
The `moi_backend == false` case seems pretty specialized to `JuMPExtension.MyModel`. IMO it would make more sense to completely split the printing tests for `JuMP.Model` and `JuMPExtension.MyModel`.